### PR TITLE
Added Background Color to Text Input

### LINKED ISSUE #4171: Add background color style in text input widget
## Proposed feature
Option to change and update the background color of the text input widget with the `Background color` property in the style tab.

### Description
Currently, the text input widget only supports **Border radius**, **Visibility**, and **Disable** as style definitions.

Screenshot 2022-10-01 at 11 13 34 AM

reference: https://github.com/ToolJet/ToolJet/pull/1721

### PR DESCRIPTION
**Pull request by rahulsunil2**
Fixes #4171

### CODE REVIEW

**Review by arpitnath — CHANGES_REQUESTED** [review]
Can you please add the default value (in the definitions) in the **widgetConfig.js**?

**Comment by arpitnath** [comment]
@rahulsunil2 You did not make any changes?

**Comment by rahulsunil2** [comment]
> @rahulsunil2 You did not make any changes?

I made that commit to merge the latest pull from the origin repo to fix the docker issue.

**Comment by arpitnath** [comment]
Okay, I have requested a few changes related to the issue. Can you please look into it 😅

**Comment by rahulsunil2** [comment]
> Can you please add the default value (in the definitions) in the **widgetConfig.js**?

I have made this change and pushed the commit.

**Comment by rahulsunil2** [comment]
@arpitnath Any updates?

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -881,6 +881,11 @@ export const widgets = [
         displayName: 'Text Color',
         validation: { schema: { type: 'string' } },
       },
+      backgroundColor: {
```
Comment (arpitnath) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:884]: We also need a default value for the same.

File: frontend/src/Editor/Components/TextInput.jsx [hunk 2]
```diff
@@ -98,7 +98,7 @@ export const TextInput = function TextInput({
           darkMode && 'dark-theme-placeholder'
         }`}
         placeholder={properties.placeholder}
-        style={{ height, borderRadius: `${styles.borderRadius}px`, color: textColor }}
+        style={{ height, borderRadius: `${styles.borderRadius}px`, color: textColor, backgroundColor: `${styles.backgroundColor}`}}
```
Comment (arpitnath) [inline frontend/src/Editor/Components/TextInput.jsx:101]: Also need to handle the background color for the default value on toggling the theme

**Review by arpitnath — CHANGES_REQUESTED** [review]

**Review by kavinvenkatachalam — APPROVED** [review]
LGTM

**Comment by withshubh** [comment]
Hey @rahulsunil2  
Thanks for contributing to ToolJet. :raised_hands: 
While this PR is under review, you can submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**